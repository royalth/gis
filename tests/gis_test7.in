6 15
1 2 
4 6
1 6
6 3
4 5 
2 4 
2 5 
6 2 
1 3 
2 3 
4 3 
3 5 
5 6
4 1 
1 5 
