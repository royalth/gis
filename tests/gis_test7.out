NIE ZNALEZIONO POKOLOROWANIA ZWARTEGO KRAWEDZI TEGO GRAFU
